src-input/duk_bi_symbol.c

```diff
@@ -35,6 +36,17 @@ DUK_INTERNAL duk_ret_t duk_bi_symbol_constructor_shared(duk_hthread *thr) {
 	 *   +1    0xff after description, before unique suffix
 	 *   +17   autogenerated unique suffix: 'ffffffff-ffffffff' is longest
 	 *   +1    0xff after unique suffix for symbols with undefined description
+   *
+   * The layout therefore looks like so:
+   *
+   * A  len B  f  f  f  f  f  f  f  f  -  f  f  f  f  f  f  f  f  C
+   * -- --- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
+   * 00 len 01 02 03 04 05 06 07 08 09 10 11 12 13 14 15 16 17 18 19
```

Comment:
This layout seems a bit confusing, the offsets in the bottom row are not continuous (due to len).